config/mapping.json

```diff
@@ -0,0 +1,137 @@
+{
```

Comment:
Yeah happy to do either, I'll import the codemod for now and that can be saprwted later if necessary.